doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -1804,3 +1804,111 @@ Example of HTTP2 logging, of a request and response:
       ]
     }
   }
+
+Event type: IKE
+---------------
+
+The following fields and example is for IKEv1, IKEv2 has a different output format. IKEv1 is denoted by the field maj_ver = 1.
+
+Fields
+~~~~~~
+
+* "init_spi", "resp_spi": The Security Parameter Index (SPI) of the initiator and responder.
+* "version_major": Major version of the ISAKMP header.
+* "version_minor": Minor version of the ISAKMP header.
+* "contained_payload_types": List of payload types in the current packet, as numeric values.
+* "contained_payload_types_verbose": List of payload types in the current packet, in human-readable form. Needs ``extended: yes`` set in the ``ike`` EVE output option.
+* "encrypted_payloads": Set to ``true`` if the payloads in the packet are encrypted.
+* "exchange_type": Type of the exchange, as numeric values.
+* "exchange_type_verbose": Type of the exchange, in human-readable form. Needs ``extended: yes`` set in the ``ike`` EVE output option.
+* "doi": Value of the domain of interpretation (DOI).
+* "server.key_exchange_payload", "client.key_exchange_payload": Public key exchange payloads of the server and client.
+* "server.key_exchange_payload_length", "client.key_exchange_payload_length": Length of the public key exchange payload.
+* "server.nonce_payload", "client.nonce_payload": Nonce payload of the server and client.
+* "server.nonce_payload_length", "client.nonce_payload_length": Length of the nonce payload.
+* "client.client_proposals": List of the security associations proposed to the server.
+* "server.chosen_proposal": The chosen security association by the server.
+* "vendor_ids": List of the vendor IDs observed in the communication.
+
+Examples
+~~~~~~~~
+
+Example of IKE logging:
+
+::
+
+  "ike": {
+    "init_spi": "e47a591fd057587f",
```

Comment:
This documentation seems inconsistent with Suricata-Verify test cf https://github.com/OISF/suricata-verify/pull/324/files#diff-934bf1defac0138e1aaeb5add9dc8bbbR19 where we use `ikev1.init_spi` or `ikev2.init_spi`...

What is the good way to go ?

Reply:
it should be ikev1, forgot to update documentation there....